packages/fast-usdc/src/update-noble-ica.core.js

```diff
@@ -50,25 +75,36 @@ harden(config);
  * @param {UpdateOpts} [config.options]
  */
 export const updateNobleICA = async (
-  { consume: { chainStorage, fastUsdcKit } },
+  { consume: { chainStorage, fastUsdcKit }, produce },
   { options = {} } = {},
 ) => {
   trace('options', options);
   const {
     agoricToNoble = config.MAINNET.agoricToNoble,
     fastUsdcCode = assert.fail('missing bundleID'),
   } = options;
-  const { adminFacet, creatorFacet, privateArgs, publicFacet } =
-    await fastUsdcKit;
-  trace('got privateArgs', Object.keys(privateArgs));
+  const kitPre = await fastUsdcKit;
+  const { adminFacet, creatorFacet, publicFacet } = kitPre;
+  trace('got privateArgs', Object.keys(kitPre.privateArgs));
 
+  const { brand: usdcBrand } = kitPre.privateArgs.feeConfig.flat;
+  // TODO: update agoricToNoble in kitPost.privateArgs.chainInfo too
```

Comment:
Does this matter? It looks like `privateArgs.(chainInfo|assetInfo)` is only consumed by ChainHub if it's the first incarnation.

Maybe worth a comment in `meta.privateArgsShape` above these two fields mentioning this.

Reply:
It's not a GTM correctness issues, but I was burned once by expecting `fastUsdcKit.privateArgs` to be current.